std/array.d

```diff
@@ -3334,6 +3368,25 @@ Appender!(E[]) appender(A : E[], E)(auto ref A array)
     });
 }
 
+///
+@safe pure nothrow
+unittest
+{
+    auto w = appender!string;
+    // pre-allocate at least 10 elements (this avoids costly reallocations)
```

Comment:
nitpick: `pre-allocate at least 10 elements` -> `pre-allocate space for at least 10 elements`